tests/integration/security/util/reachability/context.go

```diff
@@ -105,7 +105,7 @@ func (rc *Context) Run(testCases []TestCase) {
 		},
 		{
 			PortName: "tcp",
-			Scheme:   scheme.HTTP,
+			Scheme:   scheme.TCP,
```

Comment:
don't we need all combinations of TCP -> TCP, HTTP -> HTTP, TCP -> HTTP, HTTP -> TCP in order to test sniffing fully?

Reply:
This PR is the framework change including some test fixes. We can enhance the test in other PRs.

Reply:
ack